docs/guides/tipsntricks.rst

```diff
@@ -10,6 +10,28 @@ as they are developed and rediscovered.
 
 .. _blockers:
 
+
+Version Picking
+---------------
+
+Dealing with multiple releases, it's obvious that some things change from version to version. A lot
+of the time, these changes are simple, such as a string change. So that we can continue using the same
+codebase for any version, we define the idea of version picking. Version picking essentially returns
+an object depending on the version of an appliance. It's particularly useful for things like locator
+changes because most of the element handling routines are version picking away. This means if they
+receive a dict as an argument, they will automatically try to resolve it using the version picking tool.
+To use version picking is easy::
+
+    from utils import version
+
+    version.pick({'5.4': "Houses",
+                  '5.3': "House",
+                  version.LOWEST: "Boat"})
+
+In this example, if the version is below 5.3, the ``Boat`` will be returned. Anything between 5.3 and 5.4
+will return ``House`` and anything over 5.4 will return ``Houses``. There is also a ``version.LATEST``
+which points to upstream appliances.
+
```

Comment:
I love this, but think we should add a line for clarity: "Version picking matches the specified version and higher, so '5.4' would match any version higher, unless ``version.LATEST`` is included in the dict."

Also, `version.UPSTREAM` and `version.LATEST` are soon going to mean two different things (maybe!), but we can revisit this when/if that happens.